web/src/components/LikertScaleQuestionField/LikertScaleQuestionField.tsx

```diff
@@ -0,0 +1,44 @@
+import { Controller, RegisterOptions, useForm } from '@redwoodjs/forms'
+import LikertScaleQuestion from '../LikertScaleQuestion/LikertScaleQuestion'
+import type { LikertScaleQuestionProps } from '../LikertScaleQuestion/LikertScaleQuestion'
+
+interface LikertScaleQuestionFieldProps extends LikertScaleQuestionProps {
+  validation?: RegisterOptions
+  errorClassName?: string
+}
+
+const LikertScaleQuestionField = (props: LikertScaleQuestionFieldProps) => {
+  const { validation, errorClassName, ...propsRest } = props
+
+  // const { className: componentClassName, style: componentStyle } =
+  //   useErrorStyles({
+  //     errorClassName: errorClassName,
+  //     name: 'likertScaleQuestion',
+  //   })
+
+  const onSubmit = (data) => {
+    console.log(data)
+  }
+
+  const methods = useForm()
+
+  return (
+    <Controller
+      control={methods.control}
+      name="likertScaleQuestion"
+      rules={validation}
+      defaultValue={[]}
+      render={({}) => (
+        <LikertScaleQuestion
+          {...propsRest}
+          n={props.n}
+          leftHand={props.leftHand}
+          rightHand={props.rightHand}
+          question={props.question}
```

Comment:
`n`, `leftHand`, `rightHand` and `question` are already inside the `propsRest` that you spread before. See what is happening on line 11 where you de-structure the `props` object.

Reply:
Okay so I can remove it